tests/cluecode/test_copyrights_fosso.py

```diff
@@ -77,54 +78,55 @@ def build_copyright_test_methods_with_fossology_data():
 
     copyregex = re.compile('<s>(.*?)</s>', re.DOTALL | re.UNICODE)
     for expected_file, test_file in zip(expected_files, files_to_test):
-
-        expected_text = open(expected_file, 'rb').read()
+        
+        import io
+        expected_text = io.open(expected_file, 'r', encoding='utf-8').read()
```

Comment:
Since you are chaing this code, it would be best to wrap this in a proper `with ...` block to ensure descriptors are properly closed.

Reply:
Dont't understand

Reply:
1. move `import io` to the top of the module
2.  use this
```
with io.open(expected_file, 'r', encoding='utf-8') as i:
    expected_text = i.read()
```

Reply:
Why ``import io`` globally as it is not anywhere except here.